### ISSUE TITLE
[bug] `examples/04-animation_builder-deforum.json` broken

### ISSUE THREAD

**Issue by dmarx** [melMass/comfy_mtb#120]
### Describe the bug

fresh install of MTB, fully updated all nodes and packages. terminal running comfy repeatedly reports following error message:

```
got prompt                                                                                                                                                                                                      
ERROR:root:Failed to validate prompt for output 15:                                                                                                                                                             
ERROR:root:* Fit Number (mtb) 22:                                                                                                                                                                               
ERROR:root:  - Failed to convert an input value to a FLOAT value: target_max, Quart In/Out, could not convert string to float: 'Quart In/Out'                                                                   
ERROR:root:Output will be ignored                                                                                                                                                                               
Prompt executed in 0.01 seconds   
```

### Operating System

Mac

### Comfy Mode

Comfy Portable (embed) (Default)

**Comment by melMass** [comment]
I can indeed replicate...
Looking into it now

**Comment by melMass** [comment]
Fixed in #109, I'll merge it a bit later today or tomorrow in main

Thanks for the report

![670664f719](https://github.com/melMass/comfy_mtb/assets/7041726/327c2bc5-294d-4513-a65c-7c3d56efed38)
